src/mono/wasm/Makefile

```diff
@@ -72,7 +72,7 @@ ifeq ($(ENABLE_METADATA_UPDATE),true)
 endif
 
 EMCC_DEBUG_FLAGS =-g -Os -s ASSERTIONS=1 -DDEBUG=1
-EMCC_RELEASE_FLAGS=-Oz --llvm-opts 2
+EMCC_RELEASE_FLAGS=-O3 --llvm-opts 3 -flto
```

Comment:
(nit) not that it really matters, but the ordering here is different than what is in the wasm.proj file.